### ISSUE TITLE
Accessibility checklist

### ISSUE THREAD

**Issue by perpetualgrimace** [Datawheel/codelife#501]
- [ ] global — increase nav & footer link contrast
- [ ] global — add `href` to nav language links
- [ ] global — add screen reader context text to footer "learn more" link
- [ ] global — add aria landmarks
- [ ] global — add `tabindex="-1"` to `.search-reset-button` when main search is hidden
- [ ] global — add `tabindex="-1"` to map dropdown toggle (map page is more keyboard friendly)
- [ ] global — `.account-popover` is a keyboard trap; make it function like a select menu

- [ ] home — logo :focus state == .is-active state
- [ ] home — add screen reader context text to about Codelife "learn more" button

- [ ] projects — add `href` to `.project-switcher-link`s
- [ ] projects — handle `browse code blocks` toggle keyboard focus
- [ ] projects — add screen reader context text to code block island toggle buttons

- [ ] projects/slides — ace editor is missing a label
- [ ] projects/slides — code editor is a keyboard trap (unsure if this can be solved)

- [ ] codeblock/project popover — set focus to close button on open
- [ ] codeblock/project popover — code editor is a keyboard trap (unsure if this can be solved)

- [ ] profile — set `tabindex="-1"` on `.card-author-link` in projects and codeblocks (and popovers)

- [ ] profile edit — DOB input missing label
- [ ] profile edit — disable DOB datapicker on focus (keyboard trap)

**Comment by perpetualgrimace** [comment]
@alexandersimoes Doing some research into Ace editor and accessibility and it looks like we're a little screwed in two ways:
- once Ace editor has keyboard focus, it's impossible to tab through to other elements on the page
- Ace doesn't do a good job of communicating its contents to screen readers

Related (forever open) issues:
- (https://github.com/ajaxorg/ace/issues/3149)
- (https://github.com/ajaxorg/ace/issues/2164)

**Comment by perpetualgrimace** [comment]
Another issue, this time regarding Quill. The toolbar buttons have no labels, so they give screen readers no context.

Related issue from 2016:
(https://github.com/quilljs/quill/issues/1173)

Quill also had the same keyboard trap issue as Ace, but luckily we can configure it to not hijack the tab key.
